tests/monitoring/vm_monitoring.go

```diff
@@ -67,9 +67,9 @@ var _ = Describe("[Serial][sig-monitoring]VM Monitoring", Serial, decorators.Sig
 	Context("Cluster VM metrics", func() {
 		It("kubevirt_number_of_vms should reflect the number of VMs", func() {
 			for i := 0; i < 5; i++ {
-				vmi := tests.NewRandomVMI()
+				vmi := libvmi.New(libvmi.WithResourceMemory("1Mi"))
```

Comment:
Would you mind using `libvmi.NewGuestless()` from https://github.com/kubevirt/kubevirt/pull/11401?